src/utilities/index.ts

```diff
@@ -1 +1 @@
-export { convertRemEmToPx } from './convertRemToPx';
```

Comment:
this is a breaking change

Reply:
I already did that breaking change. I just forgot to rename the file.